cmd/clusterctl/cmd/describe_cluster.go

```diff
@@ -135,6 +136,7 @@ func init() {
 		"Disable grouping machines when ready condition has the same Status, Severity and Reason.")
 	_ = describeClusterClusterCmd.Flags().MarkDeprecated("disable-grouping",
 		"use --grouping instead.")
+	describeClusterClusterCmd.Flags().BoolVar(&dc.color, "color", false, "Enable color output, even when stdout is not a tty.")
```

Comment:
nit so we can use -c as well
```suggestion
	describeClusterClusterCmd.Flags().BoolVarP(&dc.color, "color", "c", false, "Enable color output, even when stdout is not a tty.")
```

Reply:
Added the shorthand and updated the example commands in the PR description.